### PR TITLE
added email verification for users

### PR DESCRIPTION
**Pull request by sahilmulla**
For #39 Add a way for users to verify their email

- This does not include email verification on sign up.

Unverified
image

Verified
image

### CODE REVIEW

File: web/admin/src/views/ProfileAccountPage.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+onBeforeMount(async () => {
+	if (props.uid === '' || props.uid === undefined) {
+		props.uid = user.data.uid
+	}
+})
+</script>
+
+<template>
+	<div class="container">
```
Comment (dherbst) [inline web/admin/src/views/ProfileAccountPage.vue:30]: Let's change this to a component, and drop the `class="container"`

**Review by dherbst — CHANGES_REQUESTED** [review]
Thanks for doing this!  There are a few changes to make before this can be merged.

**Comment by sahilmulla** [comment]
Will implement!

File: web/admin/src/views/ProfileAccountPage.vue (hunk 1 above)
Comment (sahilmulla) [reply web/admin/src/views/ProfileAccountPage.vue:30]: Should I remove the account tab?

Comment (dherbst) [reply web/admin/src/views/ProfileAccountPage.vue:30]: Yes, put the component above the registration form, something like this:

image

**Comment by sahilmulla** [comment]
I have implemented the changes

**Comment by dherbst** [comment]
Thanks for doing this!

**Comment by sahilmulla** [comment]
Glad I could help : )

**Pull request merged**